memCheck/topProcess.py

```diff
@@ -0,0 +1,26 @@
+def topProcess(numOfProcesses):
+    import psutil
+
+    ad_pids = []
+    procs = []
+    for p in psutil.process_iter():
+        with p.oneshot():
```

Comment:
Consider using oneshot for performance gains.  See here: https://psutil.readthedocs.io/en/latest/#psutil.Process.oneshot

It's less impactful since we're only grabbing the name, mem_info, pid and uss but it could save time and cpu clocks on larger calls or if we pull from many systems simultaneously.

Reply:
I believe that psutil is already taking advantage of oneshot so there is no reason to include it